tools/wptrunner/wptrunner/tests/base.py

```diff
@@ -17,7 +17,9 @@
     current_tox_env_split = os.environ["CURRENT_TOX_ENV"].split("-")
 
     tox_env_extra_browsers = {
-        "chrome": {"chrome_android"},
+        "chrome": {"chrome_android", "chrome_webdriver"},
```

Comment:
This appears to be the only change related to testing the code itself. Are there any tests we can add to Travis to keep this working?

Reply:
@gsnedders tells me that we're now running the infrastructure tests with chrome_driver now (and Selenium-based chrome) so that resolves this concern!